mifospay/src/main/res/layout/activity_splash_screen.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".SplashScreen"
+    android:layout_gravity="center"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <ImageView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@mipmap/ic_launcher_round"
```

Comment:
It will be better to use a higher resolution version of the logo. `ic_launcher_round` is a low res version suited for the app icon

Reply:
Which one would you suggest is a higher version image?

Reply:
You can obtain the original images from [here](https://drive.google.com/drive/folders/10NTOigKiTYPH0ONbIvdGvVZHrnosQNyy)